service/src/test/java/bio/terra/workspace/common/utils/MockMvcUtils.java

```diff
@@ -910,14 +909,44 @@ public ApiCloneControlledGcpGcsBucketResult cloneControlledGcsBucket(
   }
 
   /** Call cloneGcsBucket() and return immediately; don't wait for flight to finish. */
-  public ApiCloneControlledGcpGcsBucketResult cloneControlledGcsBucketAsync(
+  public ApiCloneControlledGcpGcsBucketResult cloneControlledGcsBucketWithError(
```

Comment:
Can we use these names, to be a little more consistent and succinct?

cloneControlledGcsBucketAsyncError

cloneControlledGcsBucketAsyncSuccess